### ISSUE TITLE
Error when adding logo : SyntaxError: Unexpected token '<', "<br /> <b>"... is not valid JSON

### ISSUE THREAD

**Issue by kantha88** [ellite/Wallos#106]
Hi, I installed Wallos on my Raspberry Pi through Portainer. 

I am able to add or maintain the subscription lists with the exception of logo.

Adding any logo will give error like in screenshot. 

`SyntaxError: Unexpected token '<', "<br /> <b>"... is not valid JSON`

![2024-02-09 12_14_05-Wallos - Subscription Tracker](https://github.com/ellite/Wallos/assets/5676368/b9f18524-8b29-4bd1-8702-917bb507a2cb)

Logs from portainer : 
```
127.0.0.1 -  09/Feb/2024:12:21:16 +0800 "GET /endpoints/subscription/get.php" 200
127.0.0.1 -  09/Feb/2024:12:21:21 +0800 "GET /endpoints/logos/search.php" 200
127.0.0.1 -  09/Feb/2024:12:21:23 +0800 "POST /endpoints/subscription/add.php" 200
```

I don't know if this will be relevant, but the installation is done on my home raspberry server, and I am accessing remotely through Tailscale. Please let me know if there could be any other logs or troubleshooting I could do. Please bear with me as I am familiarizing myself with linux environments.

System details
`Linux myServer 6.1.0-rpi7-rpi-2712 #1 SMP PREEMPT Debian 1:6.1.63-1+rpt1 (2023-11-24) aarch64 GNU/Linux`

**Comment by kantha88** [comment]
Update: I managed to find the response of Add.php from F12. Any help is appreciated.

```
<br/>
<b>Fatal error</b>
:  Uncaught ImagickException: WriteBlob Failed `../../images/uploads/logos/1707453706-tst.png' @ error/png.c/MagickPNGErrorHandler/1715 in /var/www/html/endpoints/subscription/add.php:53
Stack trace:
#0 /var/www/html/endpoints/subscription/add.php(53): Imagick-&gt;writeImage('../../images/up...')
#1 /var/www/html/endpoints/subscription/add.php(25): saveLogo('\xFF\xD8\xFF\xE0\x00\x10JFIF\x00\x01\x01\x00\x00...', '../../images/up...', 'tst')
#2 /var/www/html/endpoints/subscription/add.php(153): getLogoFromUrl('https://encrypt...', '../../images/up...', 'tst')
#3 {main}
  thrown in <b>/var/www/html/endpoints/subscription/add.php</b>
on line <b>53</b>
<br/>

```

**Comment by kantha88** [comment]
The error message seems to be similar to https://github.com/ellite/Wallos/issues/3, so I tried doing the fix done by ellite in the code, which is to run https://github.com/ellite/Wallos/commit/c506ff9bceb6db18e084b573e1ad15cc05a94857 manually.

It still did not fix for me the issue.

**Comment by ellite** [comment]
I was actually commenting that. It does sound like a permissions issue.

**Comment by kantha88** [comment]
Ok finally I found the issue from this issue : https://github.com/ellite/Wallos/issues/84

I followed the steps given in that issue, and found that I only had folder icons, but not logos. So I created the directory manually, granted permission as given by you, and it works perfectly now.

I am so thankful for you making the time to create the great app, and sorry for troubling you. And I too, learnt something today. :)
